root/edit/details/AddInstrument.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import EntityLink
+  from '../../static/scripts/common/components/EntityLink';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+
+type AddInstrumentEditT = {
+  ...EditT,
+  +display_data: {
+    ...DatePeriodRoleT,
+    +comment: string | null,
+    +description: string | null,
+    +instrument: InstrumentT,
+    +name: string,
+    +type: InstrumentTypeT,
+  },
+};
+
+type Props = {
+  +allowNew?: boolean,
+  +edit: AddInstrumentEditT,
+};
+
+const AddInstrument = ({allowNew, edit}: Props): React.MixedElement => {
+  const display = edit.display_data;
+  const description = display.description;
+
+  return (
+    <>
+      <table className="details">
+        <tr>
+          <th>{addColonText(l('Instrument'))}</th>
+          <td>
+            <EntityLink
+              allowNew={allowNew}
+              entity={display.instrument}
+            />
+          </td>
+        </tr>
+      </table>
+
+      <table className="details add-instrument">
+        <tr>
+          <th>{addColonText(l('Name'))}</th>
+          <td>{display.name}</td>
```

Comment:
It would look as a localization bug in non-English UI. It would be nice to specify it is intentionally untranslated (using hard-coded tag `lang="en"`) and to allow further localizing its display (using quotes or anything appropriate for the UI locale).

```suggestion
          <th>{addColonText(l('Name'))}</th>
          <td>{untranslated(display.name)}</td>
```

with something like:

```javascript
function untranslated(variable: Expand2ReactInput): Expand2ReactOutput {
  return exp.lp(
    '{untranslated_text}',
    {untranslated_text: <span lang="en">{variable}</span>},
  );
}
```

I did not test it.

Reply:
Hm, well we do display the localized name in the "Instrument:" link right above this, at least, and all other entities display untranslated names at present too. I'm not sure about a hard-coded `lang="en"` since not all instruments are in English or have English names. Though I'd be curious to hear from other translators if `{untranslated_text}` would be useful to them. @mihkel-t maybe? :) I would proceed with this in a separate PR if necessary.

Reply:
@mwiencek: I think "{untranslated_text}" on its own will only confuse translators. I'd say it needs a comment for translators, something like "Don't change the {untranslated_text} placeholder, only add e.g. quotes around it if appropriate in your language."

Reply:
Should we leave it as is for now and add a ticket like "Mark intentionally untranslated strings"?

Reply:
FWIW, I'd support adding the comment to this one right away, while it's still remembered... But if there are a whole bunch of such intentionally untranslated things, then probably a good idea to have a ticket to track them.

Reply:
IIRC we don't have a way to set comments now though?

Reply:
Would we eventually want this for all entity names, or just entity names where we have a translation but aren't using it? (If we start displaying aliases in place of entity names, that distinction might become less useful.) Just wondering because we display raw entity names in (to make an understatement) a lot of places.

And would it cause a problem if we used `{untranslated_text}` where the entity name happens to be in the UI language, so isn't actually "untranslated?"

Reply:
I kinda feel we should just have a note or warning that a field is intentionally left untranslated. Maybe an icon of some sort? https://github.com/metabrainz/musicbrainz-server/pull/1636 for example has untranslated text on edits that would look weird styled, and I agree that we shouldn't assume the text is always in English to begin with since things like instrument names are very international.

Reply:
> not all instruments are in English or have English names

Correct, many are not. We don’t have any information about the language here. Let’s forget about `lang` attribute.

> Would we eventually want this for all entity names, or just entity names where we have a translation but aren't using it? 

The latter, here we do intentionally display raw name because translations were not available at edit time.

> And would it cause a problem if we used `{untranslated_text}` where the entity name happens to be in the UI language, so isn't actually "untranslated?"

Probably not. Even with quotes or whatever around it, it would still be readable to users from that language.

> I kinda feel we should just have a note or warning that a field is intentionally left untranslated. Maybe an icon of some sort? 1636 for example has untranslated text on edits that would look weird styled

But since the only use case seems to be when creating entities, maybe we can specify it in header rather than data:
```suggestion
          <th>{addColonText(l('Name (raw)'))}</th>
          <td>{display.name}</td>
```

Reply:
I think "raw" is not too clear. What about this?

![Screenshot from 2020-08-31 15-51-14](https://user-images.githubusercontent.com/1069224/91721677-d12ef900-eba1-11ea-9d11-2893972fba20.png)

Reply:
Seems alright or just “Field value shown untranslated as entered in this edit”